### ISSUE TITLE
axI18n: add convenience for locale changes

### ISSUE THREAD

**Issue by alex3683** [LaxarJS/laxar#332]
Common use case is that a widget listens to `didChangeLocale` events for a certain locale and creates a localizer for the according language tag. This could be added as a convenience to the `axI18n` service, offering the possibility to register a callback that is called with a new localizer whenever the locale or the language tag of the locale changes.

Example:
```
axI18n.whenLocaleChanged( localizer => {
   renderAllTexts( localizer );
} );
```

**Comment by alex3683** [comment]
This should replace https://github.com/LaxarJS/laxar-patterns/blob/master/lib/i18n.js in the long run.

**Comment by x1B** [comment]
Implemented on `master` (`v2.0.0`).

### Upgrade Information

The injection `axI18n` is now available for widgets. It replaces the previous `laxar.i18n` API (only available through `laxar-compatibility` in v2.0.0) as well as the now-deprecated `i18n` helper from LaxarJS Patterns.

The most important methods are 

* `localize( i18nValue, fallback )` and `format( i18nFormatString, [ indexedReplacements], [namedReplacements] )` for localization,

* `languageTag()` and `track()` to access language tags and to track them as context properties,

* `whenLocaleChanged( callback )` for observing changes to the locale's language tag,

* `update( tag )` to modify the language tag of the current locale.

Detailed instructions are available in the *widget services* manual and in the *i18n* manual.
